pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/definition/ConsentElectionDef.java

```diff
@@ -3,46 +3,20 @@
 import javax.validation.constraints.NotBlank;
 
 import com.google.gson.annotations.SerializedName;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
 
-public class ConsentElectionDef {
-
+@Data
+@RequiredArgsConstructor
+public final class ConsentElectionDef {
     @NotBlank
     @SerializedName("stableId")
-    private String stableId;
+    private final String stableId;
 
     @NotBlank
     @SerializedName("selectedExpr")
-    private String selectedExpr;
+    private final String selectedExpr;
 
     private transient Long consentElectionId;
     private transient Long selectedExprId;
```

Comment:
This class does not have any clearly visible relation to the rest of the modifications here. Why is it being included with this PR?

Reply:
It looks like I had some Lombok-related changes and forgot to stash them. If it is a big deal, then I'll revert this change.